addons/ai/functions/fnc_assignNVG.sqf

```diff
@@ -0,0 +1,45 @@
+#include "script_component.hpp"
+/*
+ * Author: Jonpas
+ * Assigns AI first found NVG in their inventory during night time and unassigns it during day time.
+ * Calls itself periodically.
+ *
+ * Arguments:
+ * None
+
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call ace_ai_fnc_assignNVG
+ *
+ * Public: No
+ */
+
+if (!GVAR(assignNVG)) exitWith {};
+
+private _equip = sunOrMoon < 1 || moonIntensity > 0.8;
+
+{
+    private _unit = _x;
+    if (!isNull _unit && {!isPlayer _unit} && {alive _unit}) then {
```

Comment:
`alive objNull isEqualTo false`

Also not possible to get null here if it's unscheduled.
Not sure if that lazy eval is not slower.
```suggestion
    if (alive _unit && {!isPlayer _unit}) then {
```